payroll/migrations/0034_auto_20210830_1459.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 2.2.9 on 2021-08-27 15:48
+
+from django.db import migrations
+from extra_settings.models import Setting
+
+
+# flake8: noqa
+def add_donate_banner_message(apps, schema_editor):
+    donate_message_html = """<p class="lead"><strong>Dear BGA readers,</strong></p>
+        <p>First, thanks very much for visiting our Salary Database site. We know hundreds of thousands of people use it throughout the year and find it useful.</p>
+        <p><strong>But we need your help to keep this important source of information going.</strong> This database costs money and time to complete. We contract with outside organizations, and BGA staffers spend more than a year requesting, compiling, organizing and checking data from hundreds of government bodies across Illinois to bring this site to you for free. We don't run ads and we are a small nonprofit.</p>
+        <p>If every person who visited this site gave us $1, we could complete our fundraising for the year right now. If you find this database valuable, please take just a few minutes to support it. Thank you.</p>
+        <p class="text-center"><a class="btn btn-primary btn-lg" href="https://donate.bettergov.org" target="_blank">Please support this work today!</a></p>
+        """
+
+    donate_message = Setting(name='DONATE_MESSAGE',
+                             value_type=Setting.TYPE_TEXT,
+                             value_text=donate_message_html)
+
+    donate_message.save()
```

Comment:
add PAYROLL_SHOW_DONATE_BANNER as a get_or_create

Reply:
i don't know if a `get_or_create` is possible with the extra_settings plugin. i got this in the shell:

```python
>>> s = Setting.get_or_create(name='test setting', value_type=Setting.TYPE_TEXT, value_text='it is raining')
Traceback (most recent call last):
  File "<console>", line 1, in <module>
AttributeError: type object 'Setting' has no attribute 'get_or_create'
```

going to investigate the plugin

Reply:
@smcalilly `Setting.objects.get_or_create`

Reply:
oh duh, i'm missing the `Setting.objects` part

Reply:
haha jinx @hancush